addons/l10n_pk/views/res_company_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_company_form" model="ir.ui.view">
+        <field name="name">res.company.form.inherit.l10n_pk_edi</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='vat']" position="attributes">
+                <attribute name="nolabel">1</attribute>
+            </xpath>
+            <field name="vat" position="before">
+                <label for="vat" invisible="country_code == 'PK'" />
+                <label for="vat" string="NTN" invisible="country_code != 'PK'" />
```

Comment:
it's better to define vat_label in `odoo/addons/base/data/res_country_data.xml` for pk